src/test/java/com/wikia/webdriver/testcases/mobile/curatedcontenttests/MainPageTests.java

```diff
@@ -50,7 +50,7 @@ public void MercuryCuratedMainPageTest_001_CheckElementsVisibilityElementsOrderA
     navigate.toPageByPath(MobileSubpages.CC_MAIN_PAGE);
     new ArticlePageObject(driver).isFooterVisible();
 
-    List<String> elemntsWhiuchShouldBeDisplayed = Arrays.asList(AD_TOP_LEADERBOARD_SELECTOR,
+    List<String> elemntsWhichShouldBeDisplayed = Arrays.asList(AD_TOP_LEADERBOARD_SELECTOR,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "elemntsWhichShouldBeDisplayed" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)